### DISCUSSION TITLE
Clear rememberedState

### DISCUSSION THREAD

**Discussion by mhanifmuhsin** [inertiajs/inertia#982] [Help (React)] [answered]
Can you help me, how to clear rememberedState after post success ?
Thanks

**Comment by ajnsn** [comment, marked as answer]
Hey @mhanifmuhsin 
There was [a smiliar question](https://github.com/inertiajs/inertia/discussions/971) about this recently.

**Reply by mhanifmuhsin** [reply]
yes, thanks for the help @ajnsn , my problem has been solved
